### DISCUSSION TITLE
Hardware Overview | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#708] [Q&A]
# Hardware Overview | Seeed Studio Wiki

Wio-WM1110 Dev Kit Hardware Overview

https://wiki.seeedstudio.com/Get_Started_with_Wio-WM1110_Dev_Kit/

**Comment by TejaswinSingh** [comment]
Does this J-Link Debug Programmer comes with the kit? And is it necessary? Sorry, I'm new too this stuff.
